technical-reports/format/design-token.md

```diff
@@ -7,19 +7,23 @@
 ```json
 {
   "token name": {
-    "$value": "token value"
+    "$value": "#fff000",
+    "$type": "color"
   }
 }
 ```
 
+Note: The `$type` property has been added to ensure this example is valid. Please refer to the [type chapter](#type-0) for more details.
```

Comment:
Should the link be reworded to disambiguate between "5.4 Type", "6.1.2 Type", and "8 Types"?

e.g.,...

```markdown
... Please refer to the [design token type property chapter](#type-0) for more details.
```

Reply:
Agreed. I'll make that change in a moment.